### PR TITLE
Include work around for wildcards in HTTP Proxy Exclusion list

### LINKED ISSUE #1502: Include work around for wildcards in HTTP Proxy Exclusion list on GHES
### What article on docs.github.com is affected?

https://docs.github.com/en/enterprise-server@2.21/admin/configuration/configuring-an-outbound-web-proxy-server

### What part(s) of the article would you like to see updated?

HTTP Proxy Exclusion.

The current instructions say to:
> type any hosts that do not require proxy access, separating hosts with commas.

There is no mention of wildcards. However, if you enter ```.mycompany.com``` it will exclude any server ending with .mycompany.com ( this would include server.mycompany.com , web.mycompany.com, etc.mycompany.com, etc.) 

### Additional information

This was mentioned by GitHub services. I have tested it and it works for me. We have been trying to fix this issue for over 4 years.

### PR DESCRIPTION
**Pull request by mzzmjd**
### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by Highaboveus2 — APPROVED** [review]

**Comment by mzzmjd** [comment]
Link to #1501 1502

**Comment by janiceilene** [comment]
@mzzmjd Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

**Comment by brntbeer** [comment]
> @mzzmjd Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

@mzzmjd Would you mind taking a look at the pull request template (here: https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) and add it to your opening post so we can go about reviewing and merging this?

File: content/admin/configuration/configuring-an-outbound-web-proxy-server.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ When a proxy server is enabled for {% data variables.product.product_location %}
 {% data reusables.enterprise_management_console.privacy %}
 4. Under **HTTP Proxy Server**, type the URL of your proxy server.
   ![Field to type the HTTP Proxy Server URL](/assets/images/enterprise/management-console/http-proxy-field.png)
+  
 5. Optionally, under **HTTP Proxy Exclusion**, type any hosts that do not require proxy access, separating hosts with commas.
   ![Field to type any HTTP Proxy Exclusions](/assets/images/enterprise/management-console/http-proxy-exclusion-field.png)
+**Note:** For wildcard settings enter the domain preceded by a leading .  Example: .mycompany.com
```
Comment (martin389) [inline content/admin/configuration/configuring-an-outbound-web-proxy-server.md:27]:
```suggestion
**Note:** To include all hosts in a domain, you can use `.` as a wildcard prefix.  Example: `.octo-org.tentacle`
```

**Comment by martin389** [comment]
### Summary
This PR addresses https://github.com/github/docs/issues/1502

### Changes in this PR
| Article in staging  | Changes |
| --- | --- | 
“[Configuring an outbound web proxy server](https://docs-1501--patch-1.herokuapp.com/en/enterprise-server@2.22/admin/configuration/configuring-an-outbound-web-proxy-server)” | Adds a line to step 5, explaining how to use a wildcard for outbound proxy configuration.

### Tests
All tests are passing! :green_apple:

**Review by stevenans9859 — APPROVED** [review]

**Comment by martin389** [comment]
Checking this approach with the support team.

**Comment by martin389** [comment]
Support have confirmed they're happy with this approach 👍 Now working on merging this PR.

**Review by martin389 — APPROVED** [review]

**Comment by martin389** [comment]
Changes will be visible here, once published: https://docs.github.com/en/enterprise-server@2.22/admin/configuration/configuring-an-outbound-web-proxy-server

**Comment by AliAlkhtri** [comment]
AliAliAlkhtri

**Comment by TEU7ZoTHLa7sZchXmdqZn5pE75ikooVNfv** [comment]
‏‪alvandisaeed43@gmail.com‬‏

**Pull request merged**